tm-tool/textmapper/tests/org/textmapper/tool/bootstrap/bison/bison1.y

```diff
@@ -20,101 +20,6 @@
 
 %%
 
-input :
```

Comment:
unrelated change?

Reply:
It is unrelated, but it does occur whenever I run `./gen.sh`. Reverted for now.